app/assets/javascripts/lookup_keys.js

```diff
@@ -217,3 +222,60 @@ function validatorTypeSelected(item){
   var validator_rule_field = $(item).closest('.fields').find("[id$='_validator_rule']");
   validator_rule_field.attr('disabled', validatorType == "" ? 'disabled' : null);
 }
+
+const KEY_DELM = ",";
+const EQ_DELM  = "=";
+
+function match_to_key_value(match){
+  var keys = [], values = [],
+    split_matcher = match.replace(/ /g,'').split(/[,=]/);
```

Comment:
I'd have to put a space or something between them and that wouldn't work.
Is there another way to put both of them there?